controllers/reconcilers/client_helper.go

```diff
@@ -0,0 +1,65 @@
+package reconcilers
+
+import (
+	"context"
+	"reflect"
+
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+// ClientHelper includes a kube client with some additional helper functions
+type ClientHelper struct {
+	client.Client
+	SetControllerReference func(client.Object) error
+}
+
+// CreateOwned is an helper function that creates an object, sets owner reference and writes info & errors logs
+func (c *ClientHelper) CreateOwned(ctx context.Context, obj client.Object) error {
+	log := log.FromContext(ctx)
+	err := c.SetControllerReference(obj)
+	if err != nil {
+		log.Error(err, "Failed to set controller reference")
+		return err
+	}
+	kind := reflect.TypeOf(obj).String()
+	log.Info("Creating a new "+kind, "Namespace", obj.GetNamespace(), "Name", obj.GetName())
```

Comment:
, kind ? 

It seems to be copy paste in a lot of logs

Reply:
I think it's as I intend to.. example of generated log:
```
2021-12-06T15:14:52.772+0100	INFO	controller-runtime.manager.controller.flowcollector	Creating a new *v1.ClusterRole	{"reconciler group": "flows.netobserv.io", "reconciler kind": "FlowCollector", "name": "console-plugin-flowcollector", "namespace": "", "Namespace": "", "Name": "goflow-kube"}
```

Reply:
I thought the output would be the same, sorry !